BuildAddOn.py

```diff
@@ -149,7 +150,7 @@ def GetInstalledVisualStudioGenerator ():
         raise Exception ('Installed Visual Studio version not supported!')
     
 
-def GetProjectGenerationParams (workspaceRootFolder, buildPath, platformName, devKitFolder, version, languageCode, optionalParams):
+def GetProjectGenerationParams (optionalParams, workspaceRootFolder, buildPath, platformName, devKitFolder, version, languageCode):

```

Comment:
optionalParms seems like it is an optional parameter for the function. Please specify optional parameters for what - I no they are for cmake, make it obvious for other people as well.

Reply:
Renamed to additionalCMakeParams